apps/ringtones/js/actions_menu.js

```diff
@@ -139,7 +139,7 @@ ActionsMenu.prototype = {
     };
     CustomDialog.show(
       'delete-title',
-      {id: descKey, args: {tone: self._tone.name}},
+      {id: messageKey, args: {tone: self._tone.name}},
```

Comment:
Couldn't you just leave the existing strings alone and do:
```
   {id: descKey, args:  {tone: '<bdi>' + self._tone.name + '</bdi>'}}
```

I'm assuming that string changes are out of the question for 2.2, so if the above code doesn't work, I'm not sure how you can get these dialogs to display correctly.